src/main/java/stream/flarebot/flarebot/util/general/FormatUtils.java

```diff
@@ -0,0 +1,172 @@
+package stream.flarebot.flarebot.util.general;
+
+import net.dv8tion.jda.core.entities.Guild;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.Period;
+import org.joda.time.format.PeriodFormatter;
+import org.joda.time.format.PeriodFormatterBuilder;
+import stream.flarebot.flarebot.objects.GuildWrapper;
+
+import java.awt.*;
+import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.util.concurrent.TimeUnit;
+
+public class FormatUtils {
+    private static DateTimeFormatter timeFormat = DateTimeFormatter.ofPattern("MMMM yyyy HH:mm:ss");
+
+    private static final SimpleDateFormat preciseFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SS");
+
+    private static final PeriodFormatter prettyTime = new PeriodFormatterBuilder()
+            .appendDays().appendSuffix(" Day ", " Days ")
+            .appendHours().appendSuffix(" Hour ", " Hours ")
+            .appendMinutes().appendSuffix(" Minute ", " Minutes ")
+            .appendSeconds().appendSuffix(" Second", " Seconds")
+            .toFormatter();
+
+    /**
+     * Formats a duration.
+     *
+     * @param duration The duration in mils to format.
+     * @return A string that is the duration.
+     */
+    public static String formatDuration(long duration) {
+        long totalSeconds = duration / 1000;
+        long seconds = totalSeconds % 60;
+        long minutes = (totalSeconds / 60) % 60;
+        long hours = (totalSeconds / 3600);
+        return (hours > 0 ? (hours < 10 ? "0" + hours : hours) + ":" : "")
+                + (minutes < 10 ? "0" + minutes : minutes) + ":" + (seconds < 10 ? "0" + seconds : seconds);
+    }
+
+    /**
+     * Formats a String to replace {%} with the prefix.
+     *
+     * @param guild The {@link Guild} the get prefix from.
+     * @param usage The usage String.
+     * @return The String with the prefix instead of {%}.
+     */
+    public static String formatCommandPrefix(Guild guild, String usage) {
+        String prefix = String.valueOf(GuildUtils.getPrefix(guild));
+        if (usage.contains("{%}"))
+            return usage.replaceAll("\\{%}", prefix);
+        return usage;
+    }
+
+    /**
+     * Formats a String to replace {%} with the prefix.
```

Comment:
Same as above also this method shouldn't also do the same, call the other method with formatCommandPrefix(guild.getGuild(), string). You'll also need to move this above, the roots should be at the bottom.
Or make this the root one.